client/src/featureform/dashboard_metadata.py

```diff
@@ -171,7 +199,7 @@ def label_variant(variantData):
                 variantRow['status'], #status
                 {"Name":variantRow['source_name'],
                 "Variant":variantRow['source_variant']}, #source
-                variant_organiser(training_set_variant(sqlObject.query_resource( "training_set_variant", "label", labelTuple))[2]) #training sets
+                variant_organiser(training_set_variant(sqlObject.get_training_set_variant_from_label(variantRow['name'], variantRow['variant']))[2]) #training sets
```

Comment:
why 2

Reply:
[2] means the second output from the traiining_set_variant function  i think